.github/workflows/run-tests.yml

```diff
@@ -42,13 +47,22 @@ jobs:
 
     steps:
     - uses: actions/checkout@v2
+    - name: Cache chianti database
+      uses: actions/cache@v2
+      if: matrix.toxenv != 'build_docs' && matrix.toxenv != 'codestyle'
```

Comment:
This line says only run the caching on the test runs, and not the doc or codestyle runs.